File: source/clibCoreCs/Models/Context/0/_blankModel.vb

Option Explicit On
Option Strict On

Imports Contensive.Core.Controllers
Imports Contensive.Core.Controllers.genericController

Namespace Contensive.Core.Models.Context
    '
    '====================================================================================================
    ''' <summary>
    ''' Site Properties
    ''' </summary>
    Public Class _blankModel
        '
        Private cpCore As coreClass
        '
        '====================================================================================================
        ''' <summary>
        ''' new
        ''' </summary>
        ''' <param name="cpCore"></param>
        Public Sub New(cpCore As coreClass)
            MyBase.New()
            Me.cpCore = cpCore
        End Sub
        '
        '
        '
    End Class
End Namespace

File: source/Processor/Models/Domain/_BlankModel.cs

using Contensive.Processor.Controllers;
//
namespace Contensive.Processor.Models.Domain {
    [System.Serializable]
    public class _BlankModel {
        private CoreController core;
        //
        //====================================================================================================
        /// <summary>
        /// new
        /// </summary>
        /// <param name="core"></param>
        public _BlankModel(CoreController core) : base() {
            this.core = core;
        }
        //
        //
        //
    }
}